src/main/java/seedu/address/model/Model.java

```diff
@@ -79,6 +81,10 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
+    void addGithub(Index index, Github github);
```

Comment:
according to existing standards, Model is in charge of/ used to modify the AddressBook. This might violate some principles.